modules/economy.js

```diff
@@ -1,295 +1,255 @@
-                if (err) client.logError(err)
-              })
-            }
-          })
-          if (callback) callback(true)
-        } else {
-          if (callback) callback(false)
-        }
+  makeMoneyTransferToUser: (client, guild, fromUser, toUser, moneyToTransfer, callback) => {
+    module.exports.getMemberInventoryAndBalance(client, fromUser, guild, (fromUserInventoryAndBalance) => {
+      if (fromUserInventoryAndBalance.amount >= moneyToTransfer) {
+        module.exports.getMemberInventoryAndBalance(client, toUser, guild, (toUserInventoryAndBalance) => {
+          try {
+            module.exports.updateMemberBalance(client, fromUser, guild, (parseInt(fromUserInventoryAndBalance.amount) - moneyToTransfer))
+            module.exports.updateMemberBalance(client, fromUser, guild, (parseInt(toUserInventoryAndBalance.amount) + moneyToTransfer))
+          } catch (err) {
+            client.logError(err)
+          }
+        })
       } else {
-        module.exports.getDailyMoney(client, fromUser, guild, false)
-        module.exports.makeMoneyTransferToUser(client, guild, fromUser, toUser, amount, callback)
+        throw new Error('ECO_XX01')
       }
     })
   },
   fetchShopProducts: (client, guild, callback) => {
-    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ?', [guild.id], (err, rows) => {
+    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ?', [guild.id], (err, shopProducts) => {
       if (err) client.logError(err)
-      if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-        callback(rows)
+      if (shopProducts && Object.prototype.hasOwnProperty.call(shopProducts, 0)) {
+        callback(shopProducts)
       } else {
-        callback(null)
+        throw new Error('ECO_XX02')
       }
     })
   },
-  fetchShopProduct: (client, guild, product, callback) => {
-    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ? AND productName = ?', [guild.id, product], (err, rows) => {
+  getShopProduct: (client, guild, productname, callback) => {
+    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ? AND productName = ? OR productId = ? LIMIT 1', [guild.id, productname, productname], (err, rows) => {
       if (err) client.logError(err)
       if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-        callback(rows[0])
+        if (callback) callback(rows[0])
       } else {
-        client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ? AND productId = ?', [guild.id, product], (err, rows) => {
-          if (err) client.logError(err)
-          if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-            callback(rows[0])
-          } else {
-            callback(null)
-          }
-        })
+        if (callback) callback()
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
            if (callback) {callback()}
```